.palantir/revapi.yml

```diff
@@ -743,6 +743,116 @@ acceptedBreaks:
       new: "method java.util.List<org.apache.iceberg.UpdateRequirement> org.apache.iceberg.rest.requests.UpdateTableRequest::requirements()"
       justification: "Signature changed to an interface, but this is safe because\
         \ of type erasure and the original type is always returned"
+  "1.3.0":
+    org.apache.iceberg:iceberg-api:
+    - code: "java.method.addedToInterface"
+      new: "method java.lang.String org.apache.iceberg.view.ViewVersion::defaultCatalog()"
+      justification: "Acceptable break due to updating View APIs and the View Spec"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.catalog.Namespace org.apache.iceberg.view.ViewVersion::defaultNamespace()"
+      justification: "Acceptable break due to updating View APIs and the View Spec"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.view.ReplaceViewVersion org.apache.iceberg.view.View::replaceVersion()"
+      justification: "Acceptable break due to updating View APIs and the View Spec"
+    - code: "java.method.numberOfParametersChanged"
+      old: "method org.apache.iceberg.view.ViewBuilder org.apache.iceberg.view.ViewBuilder::withQuery(java.lang.String)"
+      new: "method T org.apache.iceberg.view.VersionBuilder<T>::withQuery(java.lang.String,\
+        \ java.lang.String) @ org.apache.iceberg.view.ViewBuilder"
+      justification: "Acceptable break due to updating View APIs and the View Spec"
+    - code: "java.method.removed"
+      old: "method java.lang.String org.apache.iceberg.view.ImmutableSQLViewRepresentation::defaultCatalog()"
```

Comment:
I think there's a problem: the generated Immutable classes should not be in the `api` module. All of the implementation classes for API interfaces are in core and it makes no sense to publicly expose these in API when we don't publicly expose the others.

Reply:
Immutables has a a way to configure the visibility or the package of the generated class, but unfortunately there's nothing currently available that would put the generated class into a different project :(